deliverables/README.md

```diff
@@ -8,61 +8,154 @@ To answer questions, please just put the answer on the next line after the quest
+
+**How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?**  
+Elements get their default style through "cascading."  An element's style can be defined by it's class, id, by what type of element it is (ie. p{) or by the defined value of its parent elements.  The most specific definition applies.
+
+**What are two CSS attributes you can change to push an element around on the page?**   
+Display and Position attributes move elements around a page.  Display will position the element ein relation to the elements around it, while Position will define how the element is displayed compared to the rest of the page, and then define particular positioning values.
+Element{
+  Display: inline;
+  padding: 10px;
+  }
+Element{
+  Position: relative;
+  top: 5px;
+  bottom: 5px;
+  height: 100px;
+  width:100px;
+  }
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element? **     
+CSS can be put in the HTML file via in-line:
+  Ex: <h1 style="color:red">Big Header Title</h1>
+This only changes the element it is written in line with.
+Internal:
+  <style>
+      selector{
+        property:value;
+      selector{
+        property:value;
+        }
+  </style>
+This will style all of the elements defined by the selectors in the HTML file.
+
+External:
+In a seperate CSS file.
+selector{
+  attribute:calue;
+  }
+This is ideal, as it keeps the html file clean and is more efficient.
+
+**Android Logo**  
+I did this
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**  
+Git is a platform that allows developers to collaborate and track their progress. It is especially helpful for its ability to store files as snapshots over time and work locally.
+
+**What is SCM?**  
+Software Configuration Management
+
+**What is a VCS?**  
+A VCS allows developers to see each version of their file over time.  This allows them to explore different options and recover older versions without any consequences.
+
+**Why is Git useful for a developer?**  
+Collaboration, snapshot style saving, collaboration, ability to work locally.
```

Comment:
... and branching.  You can create multiple branches, mess up one branch, and still have the other branches until you merge them back together.